setup.py

```diff
@@ -48,7 +48,7 @@
         "rich[jupyter]",
         "ruamel.yaml",
         "setuptools; python_version>='3.12'",
-        "sqlglot[rs]~=25.34.1",
+        "sqlglot[rs]~=26.0.0",
```

Comment:
I think this has created a regression in mssql, datetimes being truncated to milliseconds

Reply:
Yeah, most likely. I'm not sure yet if this is a regression or if we simply need to update the SQLMesh side to match the updated SQLGlot logic. Either way, will triage soon and patch as needed.

cc @VaggelisD

Reply:
Fixed the test as we discussed @erindru.